### ISSUE TITLE
File transfer doesn't work while using Myrtille server as a gateway.

### ISSUE THREAD

**Issue by romanad** [cedrozor/myrtille#333]
Server Win 10 x64
Myrtille 2.8.1.0

Clients: Chrome, Internet Explorer, Firefox from Windows 10 x64.

The problem is that File Transfer is greyed out if I use Myrtile server as a gateway for other PCs in local network. But if I connect to the server directly, everything works fine.

Is it normal behavior or I'm doing something wrong? 

https://i.imgur.com/LB0mehM.png

**Comment by cedrozor** [comment]
Yes, it's normal. File transfer requires the Myrtille machine to be in a domain, as it uses roaming profiles synchronization. The exception is connecting the machine itself (localhost) and a local user account.

**Comment by romanad** [comment]
Is it possible to use the workgroup instead of domain to get file transfer ability in gateway mode? Or  having a domain is a must?
